README.md

```diff
@@ -11,30 +11,53 @@ LiteFarm version 1.0.0 was released to the public in July 2020. The LiteFarm app
 
 If you’re a farmer and would like to join LiteFarm you can sign up today at app.litefarm.org. If you are a researcher or would like to find out more about this project you can contact the [UBC Centre for Sustainable Food Systems](https://ubcfarm.ubc.ca/litefarm/). If you're a developer, welcome to the team! All the details on how you can contribute to this project are right here.
 
-# Setup 
+# Setup
 
-LiteFarm is comprised of three applications which all reside in this monorepo.
+LiteFarm is comprised of three applications which all reside in this monorepo:
 
-- `packages/webapp` is the client-facing application 
-- `packages/api` is the back-end API server with entry point `src/server.js`
-- `packages/api/src/jobs` is the "jobs scheduler" for certification exports, with entry point `index.js` 
+- `packages/webapp` is the client-facing application ([read more](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1190428675/The+api+package))
+- `packages/api` is the back-end API server with entry point `src/server.js` ([read more](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1190166533/The+webapp+package))
+- `packages/api/src/jobs` is the "jobs scheduler" for certification exports, with entry point `index.js` ([read more](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1190101039/The+export+jobs+pseudo-package))
 
-## Preliminaries 
+Additionally, `packages/shared` contains utilities that both the API server and the front end code relies upon ([read more](https://lite-farm.atlassian.net/wiki/spaces/LITEFARM/pages/1221820417/The+shared+package)).
```

Comment:
Should we also add info on home folder? Some of the git cli integrations are there with lerna and husky.